pylib/Utilities/ExecuteCmd.py

```diff
@@ -209,7 +212,7 @@ def execute(self, options, cmdargs, testDef, quiet=False):
                 continue
             arg = arg.replace('\"','')
             # Look for any job names in cmdargs, and attach unique identifier
-            if arg.startswith('--job-name='):
+            if cmdargs[0] == 'srun' and arg.startswith('--job-name='):
```

Comment:
"-J" is an alternate form of --job-name

Reply:
I take care of that in a later line in which I already am checking for `srun`

https://github.com/ribab/mtt/blob/644f9a92271bb5b239ad4e9a494ea1ed773f011d/pylib/Utilities/ExecuteCmd.py#L219